### PR TITLE
Markdown conversion for `pt-BR` - Replace - Games section :warning:  Do not squash :warning: 

### PR DESCRIPTION
**Pull request by SphinxKnight**
Follow-up of https://github.com/mdn/translated-content/pull/8458

### CODE REVIEW

**Comment by SphinxKnight** [comment]
Please :warning: Do not squash merge :warning:

File: files/pt-br/games/tools/asm.js/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+translation_of: Games/Tools/asm.js
+---
+{{GamesSidebar}}{{IncludeSubnav("/en-US/docs/Games")}}
+
+[Asm.js](http://asmjs.org/) é uma especificação que define um subconjunto de JavaScript que é altamente otimizável. Esta seção examina exatamente o que é permitido no subconjunto asm.js, que melhorias confere, Onde e como você pode fazer uso dele, e mais Principais recursos e exemplos.
+
+## O que é asm.js, exatamente?
+
+##
+
```
Comment (josielrocha) [inline files/pt-br/games/tools/asm.js/index.md:13]:
```suggestion
## O que é asm.js, exatamente?

```

File: files/pt-br/games/introduction_to_html5_game_development/index.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  - : Linguagem de programação web rápida para escrever o código do seu jogo, para portar facilmente seus jogos existentes [Emscripten](https://github.com/kripken/emscripten/wiki) ou [Asm.js](http://asmjs.org/spec/latest/).
+- [Pointer Lock API](/pt-BR/docs/WebAPI/Pointer_Lock)
+  - : Bloquei o mouse ou outro dispositivo apontador na interface do seu jogo.
+- [SVG](/pt-BR/docs/SVG) (Gráficos Vetorias Escaláveis)
+  - : Crie gráficos vetoriais que sejam dimensionados sem problemas, independemente do tamanho ou da resolução da exibição do usuário.
+
+<!---->
+
+- [Typed Arrays](/pt-BR/docs/JavaScript/Typed_arrays)
+  - : Acessar dados binários brutos de dentro do JavaScript; Manipule texturas GL, dados de jogos ou qualquer outra coisa.
```
Comment (josielrocha) [inline files/pt-br/games/introduction_to_html5_game_development/index.md:59]:
```suggestion
- [SVG](/pt-BR/docs/SVG) (Gráficos Vetorias Escaláveis)
  - : Crie gráficos vetoriais que sejam dimensionados sem problemas, independemente do tamanho ou da resolução da exibição do usuário.

- [Typed Arrays](/pt-BR/docs/JavaScript/Typed_arrays)
  - : Acessar dados binários brutos de dentro do JavaScript; Manipule texturas GL, dados de jogos ou qualquer outra coisa.
```

File: files/pt-br/games/index.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  - : Um guia rápido para criar jogos 2D com HTML5, para a sua distribuição no Firefox OS e outras plataformas móveis.
+- [Gamedev.js Weekly](http://gamedevjsweekly.com/)
+  - : Boletim de notícias semanal sobre desenvolvimento de jogos HTML5, lançado toda Sexta-Feira. Contém os últimos artigos, tutoriais, ferramentas e recursos.
+- [HTML5 Game Devs Forum](http://www.html5gamedevs.com/)
+  - : Fóruns para desenvolvedores, criadores de framework e publicadores. Faça perguntas, obtenha retorno, ajude outras pessoas.
+
+<!---->
+
+- [HTML5 Game Engine](http://html5gameengine.com/)
+  - : Lista dos frameworks HTML5 mais populares para jogos, juntamente com a sua classificação, benefícios e exemplos.
```
Comment (josielrocha) [inline files/pt-br/games/index.md:61]:
```suggestion
  - : Fóruns para desenvolvedores, criadores de framework e publicadores. Faça perguntas, obtenha retorno, ajude outras pessoas.

- [HTML5 Game Engine](http://html5gameengine.com/)
  - : Lista dos frameworks HTML5 mais populares para jogos, juntamente com a sua classificação, benefícios e exemplos.
```

**Review by josielrocha — CHANGES_REQUESTED** [review]
Hi @SphinxKnight!

I have made some suggestions, could yout please check them out?

**Comment by SphinxKnight** [comment]
@josielrocha since the feedback was on "empty" content left after the conversion tool and only touched 3 documents, I've updated accordingly. Thanks a lot for the review :bow:

**Review by josielrocha — APPROVED** [review]

**Pull request merged**